dashboards/sql/immediate_needs.sql

```
@@ -1,10 +1,10 @@
-WITH totals AS (SELECT count(*) as total_count FROM analytics.mart_screener_data WHERE 1=1 [[AND {{partner}}]])
+WITH totals AS (SELECT count(*) as total_count FROM analytics.mart_screener_data WHERE 1=1 [[AND {{partner}}]] [[AND {{county}}]])
```

SQLFluff (4.0.4)

[error] Line is too long (130 > 120).

(LT05)